mlx/linalg.h

```diff
@@ -62,4 +62,6 @@ norm(const array& a, int axis, bool keepdims = false, StreamOrDevice s = {}) {
 
 std::pair<array, array> qr(const array& a, StreamOrDevice s = {});
 
+std::tuple<array, array, array> svd(const array& a, StreamOrDevice s = {});
```

Comment:
I think this output type is going to cause issues for MLX C (CC @andresy ). Could you make it a `std::vector<array>` instead?

Reply:
Done, though it's not as ergonomic because unpacking stops working.